app/pages/History/HistoryPage.presenter/HistoryPage.presenter.tsx

```diff
@@ -34,9 +35,24 @@ export const HistoryPage: FC = (): JSX.Element => {
 
   const buildList = (): TransactionLog[] => {
     if (transactionLogs) {
+
+      let changeAddressId = "None";
+      let legacyChangeAddressId = "None";
+
+      let myAddresses = Object.values(addresses.addressMap)
+      myAddresses.forEach((address) => {
+        if (address.subaddressIndex == "18446744073709551614") {
```

Comment:
"Magic" numbers should have a comment describing what they are

Reply:
Added comments! Thanks.